docs/source/library-user-guide/functions/adding-udfs.md

```diff
@@ -1047,6 +1047,48 @@ The output will be like:
 ...
 ```
 
+### Named Arguments for Window UDFs
```

Comment:
I wonder if this section on named parameters should be generic across scalar/window/aggregate functions, instead of repeating it for each?

Reply:
Sorry what do you mean by this? I was referring to the documentation

Reply:
I'm not sure if there is some kind of misunderstanding, but my comment is referring to the documentation, specifically the `adding-udfs.md` page. I'm not talking about the tests.